swift-ci/sdks/static-linux/scripts/fetch-source.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#
+# ===----------------------------------------------------------------------===
+#
+#  Swift Static SDK for Linux: Fetch Sources
+#
+#  This source file is part of the Swift.org open source project
+#
+#  Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
We should update the year

Reply:
Fixed. I've added copyright banners to various other files too.